sdk/python/pulumi_aws/acm/certificate.py

```diff
@@ -1165,7 +1165,7 @@ def not_before(self) -> pulumi.Output[str]:
 
     @property
     @pulumi.getter
-    def options(self) -> pulumi.Output[Optional['outputs.CertificateOptions']]:
+    def options(self) -> pulumi.Output['outputs.CertificateOptions']:
```

Comment:
Interesting that this type went from `Optional` to not.  It shouldn't be breaking due to being an output rather than an input or argument, though.  I expect this actually simplifies user programs since they don't need to check for `None`

Reply:
I suspect this is the outcome of it changing to a computed field in terraform-provider-aws#29763.

Reply:
Aha!  That makes a ton of sense.